browser/ui/views/sidebar/sidebar_items_contents_view.cc

```diff
@@ -567,7 +563,7 @@ ui::ImageModel SidebarItemsContentsView::GetImageForBuiltInItems(
     case sidebar::SidebarItem::BuiltInItemType::kChatUI:
       return get_image_model(kLeoProductBraveLeoIcon, state);
     case sidebar::SidebarItem::BuiltInItemType::kNone:
-      NOTREACHED_NORETURN();
+      NOTREACHED() << "we don't ask image for kNone type";
   }
 }
```

Comment:
Put another `NOTREACHED` outside the switch just in case.

Reply:
or a default fall through case?